custom_model_runner/datarobot_drum/resource/components/Python/r_fit/fit.R

```diff
@@ -24,6 +25,66 @@ init <- function(code_dir) {
 }
 
 
+load_data <- function(input_filename){
+    tmp = readChar(input_filename, file.info(input_filename)$size)
+    read.csv(text=gsub("\r","", tmp, fixed=TRUE), na.strings = c("NA", ""))
```

Comment:
I don't like this- R by default just looks for 'NA' as a null value...and nothing else